src/config/genesis_accounts.json

```diff
@@ -521,5 +394,26 @@
   },
   "0xe33BF2d6b28c6aC880cd81b5564A92a42593369d": {
     "wei": "10000001000000000000000000"
+  },
+  "0x0a0844DA5e01E391d12999ca859Da8a897D5979A": {
+    "wei": "1001000000000000000000"
+  },
+  "0x9Ce1C3c114538c625aA2488b97fEb3723fdBB07B": {
+    "wei": "1001000000000000000000"
+  },
+  "0x353Ad64Df4fAe5EffF717A1c41BE6dEBee543129": {
+    "wei": "1001000000000000000000"
+  },
+  "0xFe188788da4d41017A2aD4770502570865AF6fF7": {
+    "wei": "1001000000000000000000"
+  },
+  "0x4Ab88E00570B740c1C403034D05954584d6351d1": {
+    "wei": "1001000000000000000000"
+  },
+  "0xD815DA50966c19261B34Ffa3bE50A30A67D97456": {
+    "wei": "1001000000000000000000"
+  },
+  "0x8d404dd7bb94ba73ff7609c8ec01c01e54afae6d": {
+    "wei": "1001000000000000000000"
   }
```

Comment:
Kenny has requested that we add one more for him (genesis account with 1001 SHM): 

`0xCB65445D84D15F703813a2829bD1FD836942c9B7`

Reply:
@aniketdivekar looping you in